docs/news.rst

```diff
@@ -5,21 +5,209 @@ Release notes
 
 .. _release-VERSION:
 
-Scrapy VERSION (unreleased)
----------------------------
+Scrapy 2.13.0 (unreleased)
+--------------------------
+
+Highlights:
+
+-   Added the :reqmeta:`allow_offsite` request meta key
+
+-   HTTP/1.0 support is deprecated
+
+Modified requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+-   Dropped support for PyPy 3.9.
+    (:issue:`6613`)
 
 Backward-incompatible changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 -   The ``from_settings()`` method of
-    :class:`~scrapy.spidermiddlewares.urllength.UrlLengthMiddleware` is removed
-    without a deprecation period (this was needed because after the
-    introduction of the
+    :class:`~scrapy.spidermiddlewares.urllength.UrlLengthMiddleware`,
+    deprecated in 2.12.0, is removed earlier than the usual deprecation period
+    (this was needed because after the introduction of the
     :class:`~scrapy.spidermiddlewares.base.BaseSpiderMiddleware` base class and
     switching built-in spider middlewares to it those middlewares need the
     :class:`~scrapy.crawler.Crawler` instance at run time). Please use
     ``from_crawler()`` instead.
 
+Deprecations
+~~~~~~~~~~~~
+
+-   Functions that were imported from :mod:`w3lib.url` and re-exported in
+    :mod:`scrapy.utils.url` are now deprecated, you should import them from
+    ``w3lib.url`` directly. They are:
+
+    - ``scrapy.utils.url.add_or_replace_parameter()``
+
+    - ``scrapy.utils.url.add_or_replace_parameters()``
+
+    - ``scrapy.utils.url.any_to_uri()``
+
+    - ``scrapy.utils.url.canonicalize_url()``
+
+    - ``scrapy.utils.url.file_uri_to_path()``
+
+    - ``scrapy.utils.url.is_url()``
+
+    - ``scrapy.utils.url.parse_data_uri()``
+
+    - ``scrapy.utils.url.parse_url()``
+
+    - ``scrapy.utils.url.path_to_file_uri()``
+
+    - ``scrapy.utils.url.safe_download_url()``
+
+    - ``scrapy.utils.url.safe_url_string()``
+
+    - ``scrapy.utils.url.url_query_cleaner()``
+
+    - ``scrapy.utils.url.url_query_parameter()``
+
+    - ``scrapy.utils.url._unquotepath()``
+
+    - ``scrapy.utils.url._safe_chars`` attribute
```

Comment:
I wonder if we should remove these 2 from the notes, since they were already private.